api/src/services/import-services/critter/import-critters-service.ts

```diff
@@ -172,6 +172,10 @@ export class ImportCrittersService extends DBService {
       // SIMS payload
       simsPayload.push(critterId);
 
+      if (!row[CSVRowState]?.sexId) {
+        throw new Error('no state');
```

Comment:
Can we use the `ApiGeneralError` instead (or similar), also would be good to include the explicit issue, at least in the `errors` param.

Maybe something along the lines of: 
```
throw new ApiGeneralError('Critter import failed', ['CSVRowState.sexId is undefined'])
```

(whatever error content is the most helpful)